packages/core/src/components/forms/numericInput.tsx

```diff
@@ -201,9 +201,12 @@ export class NumericInput extends AbstractComponent<HTMLInputProps & INumericInp
             "stepSize",
         ], true);
 
+        const isLarge = className != null && className.indexOf(Classes.LARGE) >= 0;
```

Comment:
Hmm, well we already support the `pt-fill` class on `NumericInput`, so it would seem intuitive to me to support other modifiers like `pt-large` in the same way (i.e. by passing the CSS class explicitly).